README.md

```diff
@@ -222,6 +221,74 @@ Use the following options if you need to configure a private link with Coralogix
+| Kafka            | kafka.topic              | The name of the Kafka topic           |
+| MSK              | kafka.topic              | The name of the Kafka topic           |
+| Kinesis          | kinesis.event.id         | The kinesis event ID                  |
+| Kinesis          | kinesis.event.name       | The kinesis event name                |
+| kinesis          | kinesis.event.source     | The kinesis event source              |
+| kinesis          | kinesis.event.source_arn | The kinesis event source ARN          |
+| Sqs              | sqs.event.source         | The sqs event source/queue            |
+| Sqs              | sqs.event.id             | The sqs event id                      |
+| Ecr              | ecr.scan.id              | The ecr scan id                       |
+| Ecr              | ecr.scan.source          | The ecr scan source                   |
+
+Note that metadata is not added by default. You must specify the metadata keys you want in the `AddMetadata` parameter.
+
+For example, if you want to add the bucket name and key name to the log message, you would set the `AddMetadata` parameter to `s3.object.key,s3.bucket`.
+
+Some metadata keys will overlap as some integrations share the same metadata. For example, both Kafka and MSK have the same metadata key `kafka.topic` or both Kinesis and Cloudwatch metadata will be added in cases where a Cloudwatch log stream is being ingested from a Kinesis stream.
+
+##### Dynamic Subsystem or Application Name
+
+As of `v1.1.0`,you can use dynamic values for the Application and Subsystem Name parameters based on the internal metadata defined above.
+
+To do accomplish this, you can use the following syntax:
+
+```
+ApplicationName/SubsystemName: {{ metadata.key | r'regex' }}
+```
+
+For example, if you want to use the bucket name as the subsystem name, you would set the `SubsystemName` parameter to:
+
+```
+{{ s3.bucket }}
+```
+
+If you want to use the log group name as the application name, you would set the `ApplicationName` parameter to:
+
+```
+{{ cw.log.group }}
+```
+
+If you only want to use part of the metadata value, you can use a regular expression to extract the desired part. For example, If we have an `s3.object.key` value of `AWSLogs/112322232/ELB1/elb.log` and we want to extract the last part of the key as the Subsystem name, we would use the following:
+
+```
+SubsystemName: {{ s3.object.key | r'AWSLogs\/.+\/(.*)$' }}
+```
+
+This would result in a SubsystemName value of `elb.log` as this is the part of the regex that is captured by the group `(.*)`.
+
+**Important**:
+
+- The regex must be a valid regex pattern.
+- The regex must define a capture group for part of the string you want to use as the value
+- The metadata key must exist in the list defined above and be a part of the integration type that is deployed.
+
+This would extract the last part of the bucket name after the last `/`.
```

Comment:
This line is out of place. What the example does is already explained in line 282